daemon/cmd/policy_test.go

```diff
@@ -1022,356 +1002,3 @@ func (ds *DaemonSuite) testIncrementalPolicy(t *testing.T) {
 	networkPolicies = ds.getXDSNetworkPolicies(t, nil)
 	require.Empty(t, networkPolicies)
 }
-
-func TestAddCiliumNetworkPolicyV2Etcd(t *testing.T) {
```

Comment:
Looks like this test has been simply deleted rather than migrated to the policy/cell?

Reply:
This test (which did not rely on etcd at all) just called a function, which *was* migrated to TestAddCiliumNetworkPolicyByLabels.